cli/cmd/root.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/edgelesssys/era/era"
+	"github.com/spf13/cobra"
+)
+
+var eraConfig string
+var insecureEra bool
```

Comment:
move those into newRootCACmd

Reply:
same as verifyCoordinator, will be needed for other commands